workflow_array_ephys/pipeline.py

```diff
@@ -36,8 +38,15 @@ class SkullReference(dj.Lookup):
     contents = zip(['Bregma', 'Lambda'])
 
 
-# Activate "ephys" schema -----------------------------------------------------
+# Activate "ephys" schema --------------------------------------------------------------
 
 ephys.activate(db_prefix + 'ephys', 
                db_prefix + 'probe', 
                linking_module=__name__)
+
+# Import NWB export functions ----------------------------------------------------------
+
+from element_lab.export.nwb import element_lab_to_nwb_dict
```

Comment:
I started to draft this change on my fork, but my linter complained: `E402  module level import not at top of file`

Importing the export function needs to occur after the respective element has been activated, so the solve I landed on was declaring `workflow_array_ephys/export.py` with the relevant imports there. What do you think @kabilar? No need to include in this PR